uefi.go

```diff
@@ -16,6 +16,10 @@ const (
 	CentosSecVars    = "/usr/share/OVMF/OVMF_VARS.secboot.fd"
 	UnSecCodePath    = "/usr/share/OVMF/OVMF_CODE.fd"
 	UnSecVarsPath    = "/usr/share/OVMF/OVMF_VARS.fd"
+	SecCodePathAarch64      = "/usr/share/AAVMF/AAVM_CODE.ms.fd"
```

Comment:
```suggestion
	SecCodePathAarch64      = "/usr/share/AAVMF/AAVM_CODE.secboot.fd"
```

Reply:
I believe this path is common on ubuntu and centos, I'd expect it to be common on arm64 as well.